packages/designsystem/tailwind.config.js

```diff
@@ -24,6 +24,7 @@ module.exports = {
         'warning-lighten': 'var(--color-warning-lighten)',
         'warning-light': 'var(--color-warning-light)',
         'warning-dark': 'var(--color-warning-dark)',
+        lck: 'var(--text-color)',
```

Comment:
Could you rename it with a more understandable name, like `text` ?

Reply:
When we use it, it will be `text-color-lck`.
It would be strange to rename it `text-color-text`, wouldn't it?

Reply:
See #224